django/core/serializers/json.py

```diff
@@ -62,7 +62,7 @@ def end_object(self, obj):
 
     def getvalue(self):
         # Grand-parent super
-        return super(PythonSerializer, self).getvalue()
+        return super().getvalue()
```

Comment:
This is an occurence of what @aaugustin mentioned in #7875 and why this change cannot be automated by a simple search and replace.